AEPMessaging/Sources/Messaging.swift

```diff
@@ -183,16 +237,43 @@ public class Messaging: NSObject, Extension {
             // either this isn't the type of response we are waiting for, or it's not a response for our request
             return
         }
-        
+
         // if this is an event for a new request, purge cache and update lastProcessedRequestEventId
         var clearExistingRules = false
         if lastProcessedRequestEventId != event.requestEventId {
             clearExistingRules = true
-            lastProcessedRequestEventId = event.requestEventId
+            lastProcessedRequestEventId = event.requestEventId ?? ""
+        }
+
+        // parse and load message rules
+        Log.trace(label: MessagingConstants.LOG_TAG, "Loading in-app or feed message definitions from personalization:decisions network response.")
```

Comment:
thinking out loud here - wonder if we should try to make this log more generic with "inbound" verbiage as opposed to specifically listing the definitions that might be retrieved?

Reply:
IMO "inbound" might not be popular and probably niche. I believe `Loading message definitions...` should suffice.